generic_abs.xml.fdm_material

```diff
@@ -25,6 +25,7 @@ Generic ABS profile. The data in this file may not be correct for your specific
         <setting key="standby temperature">200</setting>
         <setting key="adhesion tendency">0</setting>
         <setting key="surface energy">70</setting>
+        <setting key="shrinkage ratio percentage">0.9</setting>
```

Comment:
Please rename this to `shrinkage percentage`. It is not a ratio. Or rename it to `shrinkage ratio` and make it an actual ratio (so this value would become `0.994`).